features/exploredash/src/main/java/org/dash/wallet/features/exploredash/ExploreDatabase.kt

```diff
@@ -45,7 +46,7 @@ import kotlin.coroutines.resumeWithException
     MerchantFTS::class,
     Atm::class,
     AtmFTS::class
-], version = 1)
+], version = 2, exportSchema = true)
```

Comment:
exportSchema is used mainly to properly configure the DB generated by the Explore Sync Tool.  We need the `identity_hash`